tools/depth-quality/depth-metrics.h

```diff
@@ -175,13 +175,10 @@ namespace rs2
                     {
                         // units is float
                         float pixel[2] = { float(x), float(y) };
-                        float point[3];
                         auto distance = depth_raw * units;
 
-                        rs2_deproject_pixel_to_point(point, intrin, pixel, distance);
-
                         std::lock_guard<std::mutex> lock(m);
-                        roi_pixels.push_back({ point[0], point[1], point[2] });
+                        roi_pixels.push_back({ pixel[0], pixel[1], distance });
```

Comment:
Later we call `plane p = plane_from_points(roi_pixels);`
and we changed the points to pixel, how does this affect the logic?